python/lsst/analysis/tools/atools/diffMatched.py

```diff
@@ -115,86 +134,260 @@ def setDefaults(self):
         self.vectorKey = "refcat_is_pointsource"
 
 
-class MatchedRefCoaddToolBase(MagnitudeXTool):
+class InjectedObjectSelector(SelectorBase):
+    """A selector for injected objects."""
```

Comment:
Why is this selector being defined in the atool and not in vector actions?

Reply:
No particularly compelling reason.